src/components/MegaButton/__tests__/MegaButton.spec.js

```diff
@@ -0,0 +1,125 @@
+import '@testing-library/jest-dom';
+import { render, fireEvent } from '@testing-library/vue';
+import MegaButton from '../MegaButton.vue';
+
+let label;
+let props;
+
+describe('Megabutton', () => {
+
+  describe('Megatext format', () => {
+
+    const megatextProps = {
+      id: 'megabutton',
+      modelValue: '',
+      name: 'Chonky Boi',
+      label: 'Megabutton',
+      value: 'bigOlButton',
+      disabled: false
+    };
+
+    it('disables the input when disabled prop is true', () => {
+      props = {
+        ...megatextProps,
+        disabled: true
+      };
```

Comment:
Instead of mutating the same `props` object and reusing for each test it would be safer/cleaner to just create a new `const props = {}` object isolated to each test.  We don't gain anything by reusing the variable but it does add the ability to screw up subsequent tests if we're not careful with how we're mutating the value.

Reply:
Fair enough.